File: algorithm/#0832/code.py
class Solution(object):
    def flipAndInvertImage(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        reta=[]
        for ai in A :
            retai=ai[::-1]
            for i in range(len(retai)) :
                if retai[i] == 0 :
                    retai[i]=1
                else :
                    retai[i]=0
            reta.append(retai)
        return reta
        

solution=Solution()
print(solution.flipAndInvertImage([[1,1,0,0],[1,0,0,1],[0,1,1,1],[1,0,1,0]]))

File: algorithm/#0263/code.cpp
class Solution {
public:
    bool isUgly(int n) {
        int a=n;
        if(a<1) return false;
        while(a%2==0) a/=2;
        while(a%3==0) a/=3;
        while(a%5==0) a/=5;
        if(a==1) return true;
        else return false;
    }
};
